python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
+    MAX_DISPLACEMENT = 10**7
+
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        self._info_script = {
+            "file": create_label(""),
+            "progress": self._create_progress_bar(0, 100, 0),
+        }
+
+        self._labels_force = {
+            "axial_min": create_label(""),
+            "axial_max": create_label(""),
+            "axial_total": create_label(""),
+            "tangent_min": create_label(""),
+            "tangent_max": create_label(""),
+            "tangent_total": create_label(""),
+        }
+
+        self._buttons_script = {
+            "load_script": set_button("Load Script", self._callback_script_load_script),
+            "run": set_button("Run", self._callback_script_command, CommandScript.Run),
+            "stop": set_button(
+                "Stop", self._callback_script_command, CommandScript.Stop
+            ),
+            "pause": set_button(
+                "Pause", self._callback_script_command, CommandScript.Pause
+            ),
+            "resume": set_button(
+                "Resume", self._callback_script_command, CommandScript.Resume
+            ),
+            "clear": set_button(
+                "Clear", self._callback_script_command, CommandScript.Clear
+            ),
+        }
+
+        self._target_displacement = self._create_target_displacement()
+        self._displacement_unit_selection = self._create_displacement_unit_selection()
+
+        self._ring_selection = self.create_combo_box_ring_selection()
+        self._buttons_actuator_selection = self._create_buttons_actuator_selection()
+        self._buttons_actuator_selection_support = {
+            "select_ring": set_button("Select Ring", self._callback_select_ring),
+            "clear_selection": set_button(
+                "Clear Selection", self._callback_clear_selection
+            ),
+        }
+
+        self._buttons_actuator = {
+            "start": set_button("Start", self._callback_actuator_start),
+            "stop": set_button(
+                "Stop", self._callback_actuator_command, CommandActuator.Stop
+            ),
+            "pause": set_button(
+                "Pause", self._callback_actuator_command, CommandActuator.Pause
+            ),
+            "resume": set_button(
+                "Resume", self._callback_actuator_command, CommandActuator.Resume
+            ),
+        }
+
+        self._set_widget_and_layout()
+
+        self._set_signal_script(self.model.signal_script)
+        self._set_signal_detailed_force(
+            self.model.utility_monitor.signal_detailed_force
+        )
+
+    def _create_progress_bar(self, value_min, value_max, value_init):
```

Comment:
This can be safely dropped, as you don't need any special ProgressBar to show 0-100 range. See above